### DISCUSSION TITLE
Reso Coder’s Flutter Architecture course

### DISCUSSION THREAD

**Discussion by MoyinShabi** [Tomic-Riedel/Flutter-Roadmap#7] [General]
Hi, @Tomic-Riedel. Really great and helpful repo this is! Thank you! I wanted to ask that is Reso coder’s TDD architecture course on YT really still fine to go though even though it’s outdated. I’m asking because i saw a number of comments talking about how it’s outdated and not recommended to go through. Thanks in advance for your response.

**Comment by Tomic-Riedel** [comment]
Hello!
First of all, thank you for liking this repository.

Your question is definitely justified and I can only give you my opinion here:
I personally still use this architecture and modify it a bit depending on what the project "demands".

Even if this architecture is outdated and maybe should not be learned as "Main Architecture", I think it gives the user a very good start into the world of Folder Structure. If the learner wants to develop his own Structure later, he knows what he has to consider at least.

But: I will definitely update this part, because I want to link as up to date knowledge as possible.
Thanks a lot for your hint!

**Comment by MoyinShabi** [comment]
Thanks for your response. I do understand your point and i’ll still check the playlist out to learn the good folder structure. I await the update also, thank you.

**Reply by Tomic-Riedel** [reply]
Hello Moyin!
I have now added a second architecture to the repository that you can learn. 
I haven't tested it myself yet, but it sounds very logical and clean.

Have fun learning!

**Reply by MoyinShabi** [reply]
Ouu nice, thank you!
